packages/forms/resources/js/components/file-upload.js

```diff
@@ -784,28 +795,39 @@ import zh_CN from 'filepond/locale/zh-cn'
 import zh_TW from 'filepond/locale/zh-tw'
 
 const locales = {
+    am,
     ar,
+    az,
     ca,
     ckb,
     cs,
     da,
     de,
+    el,
     en,
     es,
+    et,
```

Comment:
This language doesn't have any translations in Filament at the moment, so please remove it

Reply:
Ok, it's been removed.